ddtrace/contrib/pytest/plugin.py

```diff
@@ -115,19 +94,14 @@ def patch_all(request):
 
 @pytest.hookimpl(tryfirst=True, hookwrapper=True)
 def pytest_runtest_protocol(item, nextitem):
-    pin = Pin.get_from(item.config)
-    if pin is None:
+    if not CIRecorder.enabled:
         yield
         return
-    with pin.tracer.trace(
+    with CIRecorder._instance.tracer.trace(
```

Comment:
Good to either clarify this or change the interface a bit. Why not use a `Pin`?

Reply:
Added more details to PR description.